### ISSUE TITLE
Mustard Seed Ventures-Alex Pitt

### ISSUE THREAD

**Issue by jaahmuhl** [promazo/Content-Team#49]
Transcript: https://www.rev.com/transcript-editor/shared/25jDWOKysraWJ5OPnemZGnnRWGWt3dHD0YBLfej6z3FyFcyLh73EWiD_J0DXI7Rx_fd7O3Ttrxh6Cz7vO9FPpXABJCs?loadFrom=SharedLink

**Comment by jaahmuhl** [comment]
Link to Content: https://drive.google.com/drive/folders/1IpXUdo9aYcfSnhVolPKSQmRXJo5WqBBp

**Comment by ArseniySky** [comment]
INAUDIBLE (PLEASE CUT):

39:37 - 40:01
40:20 - 41:12
43:48 - 44:16
45:57 -  end

**Comment by ArseniySky** [comment]
MEDIUM LENGTH

Caption:
Alex Pitt, Co-Founder of Mustard Seed Ventures, has succeeded as a student, as a corporate worker, and as an entrepreneur. Alex has a resume like no other and talks about his struggles transitioning and starting his own business. Funny enough, Mustard Seed Ventures started with students pitching ideas to Alex and has blown up ever since. However, it hasn't always been smooth sailing and as you can imagine, Alex has the stories to back it up.

REDONE HOOK.

Q: 2:38 - 2:49 (11 seconds)
A: 2:50 - 4:20 (90 seconds)

Q: 6:44 - 6:51 (7 seconds)
A: 7:02 - 8:08 (66 seconds)

Q: 10:57 - 11-13 (16 seconds)
A: 11:15 - 11:32, 11:59 - 12:24

Q: 36:07- 36:45 (35 seconds)
A: 36:46 - 37:30 (44 seconds)

Q: 37:31 - 37:48, 38:01 - 38:07 (23 seconds)
A: 38:08 - 38:52 (44 seconds)

Total = 581 seconds, 9 minutes 41 seconds

Interviewers: Brett Hummel
Videographer: Bryan Acevedo

#promazo #reallifemba #promazoexecinterviews #mustardseed #networking #VC #entrepreneurship #grades #experience #relationships

**Comment by ArseniySky** [comment]
SHORT LENGTH

Caption:
Want to know how to land your first job? Well don't take it from us, take it from Alex Pitt, the Co-Founder of Mustard Seed Ventures!

Q: 10:57 - 11-13 (16 seconds)
A: 11:15 - 11:32, 11:59 - 12:24 (42)

Total = 58 sec

Interviewers: Brett Hummel
Videographer: Bryan Acevedo

#promazo #reallifemba #promazoexecinterviews #mustardseed #networking #VC #entrepreneurship #grades #experience #relationships

I DONT THINK THIS IS THE BEST THING FROM HIS INTERVIEW. HE REALLY ISNT SAYING ANYTHING VERY INSIGHTFUL

**Comment by ArseniySky** [comment]
LONG LENGTH

Caption:
Alex Pitt, Co-Founder of Mustard Seed Ventures, a principal investor in world-class early-stage businesses joins us to talk about the importance of networking and connecting your personal with your professional connections. He also explains what it takes to land your first job out of college and even how to start your own company. Alex has been a student, worked in the corporate world, and even started his own VC firm. Understanding where you are right now, and where you want to be in the future is key to getting on the right path for you and we dive into how Alex found his path to showcase how it can be a pretty twisting road to get where you eventually want to be.

Intro:
Clip: 
38:17 - 38:52
Description of the overall podcast (same sentences):
Hello everyone and welcome back to Executives Uncensored where we talk to the executives you want to hear from about the things we don’t know enough about. We know that landing your first job is a complicated, stressful and uncertain campaign, but not impossible. We're here to talk with leaders from across the business world to get their advice on how to build a successful career. Take what you can, pick what you like and listen for that stroke of insight that will help get you hired. 

Story:
So as we get into this interview, I'm not sure if anyone ever told you, but entrepreneurship is hard. It can’t be a part time job that you do on the side, or even a 9-5 that you do instead of a regular job. It takes all day, and even many nights at the start of it all. We often hear about all the success stories and how cool it is to call yourself CEO but you don’t read about the failures, the torn friendships between the co-founders and the endless rejections. You don’t hear about the missed Friday nights and vacation plans because you have to reorganize the excel sheet and make some cold-calls at 7:30. We put entrepreneurship on a pedestal, praise those that have done it successfully and assure ourselves that we too, will do it soon, when the time is right, when the stars align, when we have enough capital or established ourselves solidly enough. We love to kick the can down the road. But who can blame us? Every idea is different, and far from everyone can be an Airbnb, an Uber or an Apple. The really scary part is that even if we came up with the idea of Facebook and we could code like Zuckerberg, there is absolutely zero guarantee that the result would be the same. So who can blame us for wanting to get a real job first, or waiting to save up some money and buy a house? No one can blame you; but no one will tell you to do it either. Entrepreneurship is a leap of faith, a trust in yourself, your team and the idea. It might work and it might not, but it certainly did for Alex Pitt 

Quick How Alex went from Point A to Point B:
Alex started just like us, a student. He went to LSE or London School of Economics where he majored in Government and Economics. He did what many of us who are in finance or CS hope to do: get a job with Goldman Sachs in London before transitioning to the Boston Consulting Group and having the opportunity to travel the world from Chicago to Dubai. However, as he was progressing in the corporate world, he started to realize that he was slowly enjoying it less and less, eventually deciding that it was time for a change and that he wanted to become an entrepreneur. And that's where the hard part came in. Even though he partnered with a good friend from LSE, they couldn't think of a good idea, which tends to be a critical component if you hope to become a successful entrepreneur. Undeterred, they decided to look at entrepreneurship from a different angle, and they realized they were better at helping others organize their ideas than creating their own so they decided to open their own investment fund. Their initial goal was just to help students who hoped to become entrepreneurs and with his classmate Henry Wigan, Alex came back to his alma mater to meet some of the more ambitious students through a pitch competition. They put out a simple flyer on campus and they were only expecting maybe 20 people to show. Instead over 150 students came to speak with them and pitch their ideas. Impressed with the caliber of students and ideas, Alex and Henry went to another university, then another and another, helping as many students as they could… They did this for 18 months, traveling across Europe to help entrepreneurs and give them initial funding all while working their day-jobs as a side hustle until they both realized this was their passion.

Bio on the individual/company:
Eventually though they realized they had to go all in and so they formed their fund, Mustard See Ventures and also quit their high paying jobs. Since 2015  Mustard Seed has become a principal investor in world-class early-stage businesses. Most importantly, Mustard Seed invests in firms that not only drive impactful growth but they have a rather unique an investment thesis that requires their companies to have a social good component of their businesses to ensure each of their investments has an impact on humanity. And with over 20 investments made already, Mustard Seed is quickly growing in the European venture capitalism scene.

Transition to podcast:
Alex made Mustard Seed not only because he wanted to be an entrepreneur, but also because he believes in investing into the world to help generations to come. He believes in planting the mustard seed now so the tree can grow later. 

Interviewers: Brett Hummel, Howie Emond
Videographer: Bryan Acevedo

#promazo #reallifemba #promazoexecinterviews #mustardseed #networking #VC #entrepreneurship #grades #experience #relationships

INAUDIBLE (PLEASE CUT):

39:37 - 40:01
40:20 - 41:12
43:48 - 44:16
45:57 -  end

**Comment by ArseniySky** [comment]
SHORT LENGTH

Caption:
Times are changing and people are switching careers more often than ever before. If you don't know what you want to do yet, or if you want to make a change, no better time than the present according to Alex Pitt.

Alex Pitt, Co-Founder of Musturd Seed Ventures, has succeeded as a student, as a corporate worker, and as an entrepreneur. Alex has a resume like no other and talks about his struggles transitioning and starting his own business. Funny enough, Musturd Seed Ventures started with students pitching ideas to Alex and has blown up ever since. However, it hasn't always been smooth sailing and as you can imagine, Alex has the stories to back it up.

A: 16:43 - 17:04 (21)

Total = 21 sec

Interviewers: Brett Hummel
Videographer: Bryan Acevedo

#promazo #reallifemba #promazoexecinterviews #mustardseed #networking #VC #entrepreneurship #grades #experience #relationships

**Comment by ArseniySky** [comment]
SHORT LENGTH

Caption:
People can make or break a company and Alex Pitt is the first to admit that. Choosing the right people to work alongside you as you start a business is absolutely crucial to the business's success, especially when the person in question is your best friend.

Alex Pitt, Co-Founder of Musturd Seed Ventures, has succeeded as a student, as a corporate worker, and as an entrepreneur. Alex has a resume like no other and talks about his struggles transitioning and starting his own business. Funny enough, Musturd Seed Ventures started with students pitching ideas to Alex and has blown up ever since. However, it hasn't always been smooth sailing and as you can imagine, Alex has the stories to back it up.

A: 29:47 - 30:25 (38)

Total = 38 sec

Interviewers: Brett Hummel
Videographer: Bryan Acevedo

#promazo #reallifemba #promazoexecinterviews #mustardseed #networking #VC #entrepreneurship #grades #experience #relationships

**Comment by jaahmuhl** [comment]
@bryanace53 So I had a chance to go through the podcast today. I agree we really need to work on the music. It doesn't fit at all. Is there anything online to help you find something? Can we get a friend to record something for us? This is especially true during the later parts of the episode where the music comes in. I realize that you mentioned this but we do need to figure out a solution for it because it doesn't work in the current form.

Then a few things with the podcast itself. I think the person introducing the podcast should say our opening about the podcast and then there should be a pause then the introduction (again listen to the NPR podcast). I realize this part was not in your control, but I wanted to point this out for when you and Arseniy work on finalizing it this week. I am guessing it will take multiple times until he can get the pacing of the introduction right so please plan accordingly for this.

Also there are multiple times where the audio really breaks which I feel like should have already been cut out. These times are approximate but it happens around 23:25 or 25:23 (not sure which), 38:25, 41:40 and 43:04. 

Additionally Arseniy had pointed out times in the first post to cut (I don't know if you did that or not). And for example there is a point where I tell the interviewee to repeat his answer because he broke up. I don't know why that wasn't cut from it.

**Comment by jaahmuhl** [comment]
@bryanace53 So I had a chance to go through the podcast today. I agree we really need to work on the music. It doesn't fit at all. Is there anything online to help you find something? Can we get a friend to record something for us? This is especially true during the later parts of the episode where the music comes in. I realize that you mentioned this but we do need to figure out a solution for it because it doesn't work in the current form.

Then a few things with the podcast itself. I think the person introducing the podcast should say our opening about the podcast and then there should be a pause then the introduction (again listen to the NPR podcast). I realize this part was not in your control, but I wanted to point this out for when you and Arseniy work on finalizing it this week. I am guessing it will take multiple times until he can get the pacing of the introduction right so please plan accordingly for this.

Also there are multiple times where the audio really breaks which I feel like should have already been cut out. These times are approximate but it happens around 23:25 or 25:23 (not sure which), 38:25, 41:40 and 43:04. 

Additionally Arseniy had pointed out times in the first post to cut (I don't know if you did that or not). And for example there is a point where I tell the interviewee to repeat his answer because he broke up. I don't know why that wasn't cut from it.

**Comment by bryanace53** [comment]
Here is the podcast just putting it on Zenhub so its easy to grab. The others I will add in the Medium and Short in a folder.

**Comment by bryanace53** [comment]
https://drive.google.com/drive/u/4/folders/1T7Mpy7n_vQyhMBE2_GyA5kduuNyeArDV

**Comment by bryanace53** [comment]
Also going to re-watch in case I missed anything.

**Comment by jaahmuhl** [comment]
@bryanace53 This links to Caterpillar and it is supposed to be to Alex Pitt.

**Comment by bryanace53** [comment]
https://drive.google.com/drive/u/4/folders/1FnP7zi7nmM0MSQEb_OeQryBWEjaF33xL

**Comment by bryanace53** [comment]
Here is the link @jaahmuhl 

https://drive.google.com/drive/u/4/folders/1FnP7zi7nmM0MSQEb_OeQryBWEjaF33xL

**Comment by bryanace53** [comment]
https://drive.google.com/drive/u/4/folders/1BY8ZBrDxVsrEjp3wva48LhTGSt63rSmE 

Intros will be in here

**Comment by jaahmuhl** [comment]
@bryanace53 

For the first song the clip (where Alex is talking) we had talked about putting the downbeat portion of the song from 00:48-00:56 as a constant loop until he finishes. I would also like to start the song before he starts talking (look at https://www.npr.org/2022/02/11/1080195179/telfar-telfar-clemens-and-babak-radboy for how they play a couple of chords before they start to talk).

Then once the clip is finished give a little (very little) break and start from the beginning of the song. Again reference the NPR piece. Also I think it would be good to fade out his audio as well.

**Comment by bryanace53** [comment]
Sounds good I rewatched again let me know if this is how you wanted it. I did the fades on the both the guys. Had a couple of notes at the beginning and then had the beat drop on the second part of the song.

**Comment by bryanace53** [comment]
https://drive.google.com/drive/u/4/folders/1BY8ZBrDxVsrEjp3wva48LhTGSt63rSmE

**Comment by jaahmuhl** [comment]
@bryanace53 

1. Remove the cymbol sound at 00:08 and throughout the rest of the podcast
2. At the end of the clip 00:39 let’s see if we can have a slightly faded out voice and then give a 2 second pause before going into the hype part of the song. However, one thing that is happening right now is there is a hard break between the lower song and the hype song. You’ve got to smooth this out and you can do this by timing the lower beat part of the song with the hype beginning part (look at NPR from 00:38-00:40 https://www.npr.org/2022/02/11/1080195179/telfar-telfar-clemens-and-babak-radboy) to see how they are doing it.
3. At 00:54 we need to pause for four seconds with the music still playing maybe a little higher volume. See the NPR podcast and how they do it from 00:53-00:57
4. After the new 4 second pause we are going to insert a new intro that introduces the person we are talking about. I will have Arseniy record and submit this to you. During this new intro 2 seconds before Arseniy finishes, the music should fade out. Look at 1:07-1:10 on the NPR podcast to see how they do this
5. After that new intro we want to have another 5 second pause before going into the second song. 1.5 seconds after the start of the second song, start Arseniy again from 00:55 in our current podcast with the new song playing on loop. See 1:10-1:18 in the NPR podcast for how it should work. I believe the host also finishes what they are saying and they let the music play
6. After completing the podcast intro, we want to have a one second pause where the music still continues underneath and then it launches with the interviewee speaking. Two timing options for second song (basement by Homegrown). Either 00:00-00:10 or 00:48-1:15. Whichever you think works best. See NPR podcast 3:54-3:56
7. Continue the second song while the interviewee starts and have the second song go underneath for about 17 seconds and then fade out. See 3:56-4:13 on the NPR podcast.

**Comment by bryanace53** [comment]
Sounds good I reviewed what you need me to do. But I think I am waiting for the updated audio.

**Comment by bryanace53** [comment]
https://drive.google.com/drive/u/4/folders/1BY8ZBrDxVsrEjp3wva48LhTGSt63rSmE

Here is the test intro not all the edits from above are done. I am still waiting for Arseniy's new intro let me know when you can get it for me. Also to be honest this is a lot for an intro we previously discuss to work our way up to NPR level. I think there are some things we can do like NPR but I think we should work our way up so we don't prolong this intro and podcast series.

**Comment by ArseniySky** [comment]
I am not certain if you are referring to a new intro altogether or if you are unaware that I recorded a new one on July 20th.

**Comment by jaahmuhl** [comment]
@bryanace53 

Please see the V3 at the bottom of the doc regarding how to set up the podcast and we can discuss when you get back.

https://docs.google.com/document/d/1TToDYSeyRpDDk9AxfMLZdHUmftpj0et-3XU57HK-glI/edit

**Comment by bryanace53** [comment]
https://drive.google.com/drive/u/4/folders/1BY8ZBrDxVsrEjp3wva48LhTGSt63rSmE

Try this^

**Comment by bryanace53** [comment]
https://drive.google.com/drive/u/4/folders/1BY8ZBrDxVsrEjp3wva48LhTGSt63rSmE

Updated

**Comment by bryanace53** [comment]
https://drive.google.com/drive/u/4/folders/1BY8ZBrDxVsrEjp3wva48LhTGSt63rSmE

Updated Video

**Comment by bryanace53** [comment]
https://drive.google.com/drive/u/4/folders/1BY8ZBrDxVsrEjp3wva48LhTGSt63rSmE

**Comment by bryanace53** [comment]
https://drive.google.com/drive/u/4/folders/1BY8ZBrDxVsrEjp3wva48LhTGSt63rSmE

**Comment by jaahmuhl** [comment]
@bryanace53 

DECEMBER PODCAST NOTES:

1. I have my volume at 100 on my audio player,  and master volume was also at 100 on my device.   I think the whole mix needs to be a little louder over all to be able to translate on different mediums.   Take the master fader and give it like a +3-5db boost.   This will insure that the listener will have maximum volume and they can  always turn their device down if they need to.
2. Cut the silence in the beginning of the podcast from where it currently is 00:38-00:42 to 00:38 to 00:40.
3. At 06:04 there is a pop in the audio when I first come in. There shouldn’t be the pop

**Comment by bryanace53** [comment]
https://drive.google.com/drive/u/4/folders/1BY8ZBrDxVsrEjp3wva48LhTGSt63rSmE
